DependencyInjection/JsonLDClientExtension.php

```diff
@@ -33,15 +36,21 @@ public function load(array $configs, ContainerBuilder $container)
         $container->register(MappingCollection::class, MappingCollection::class)
             ->setFactory(array(MappingCollection::class, 'create'))
             ->addArgument($config['mappings'])
+            ->addArgument($config['api_root_url'] ?? '')
             ->setPublic(false);
 
-
         $loader = new YamlFileLoader(
             $container,
             new FileLocator(__DIR__ . '/../Resources/config')
         );
 
+        $defaultNamespace = $config['default_class_namespace'] ?? '';
```

Comment:
This doesn't work. We had this config option in the past but the problem occur straight away when using multiple APIs which both backoffice and premium does.